test/harvester/sse.js

```diff
@@ -0,0 +1,91 @@
+var $http = require('http-as-promised');
+var harvester = require('../../lib/harvester');
+var baseUrl = 'http://localhost:' + 8002;
+var chai = require('chai');
+var expect = chai.expect;
+var ess = require('event-source-stream');
+var _ = require('lodash');
+
+describe('EventSource implementation for resource changes', function () {
+
+    describe('Server Sent Events', function () {
+      this.timeout(100000);
+      var lastEventId;
+      var lastDataId;
+      
+      before(function (done) {
+
+        var that = this;
+
+        var options = {
+            adapter: 'mongodb',
+            connectionString: process.argv[2] || process.env.MONGODB_URL || "mongodb://127.0.0.1:27017/test",
+            db: 'test',
+            inflect: true
+        };
+
+        that.harvesterApp =
+          harvester(options)
+          .resource('book', {
+              title: String
+          });
+
+        that.harvesterApp.listen(8002);
```

Comment:
I didn't really to how to shut it off, 8001 was occupied too, someone else has been there